include/modem/pdn.h

```diff
@@ -221,13 +221,22 @@ int pdn_default_apn_get(char *buf, size_t len);
  * @brief Register a callback for events pertaining to the default PDP context (zero).
  *
  * @param cb The PDN event handler.
+ *
+ * @retval 0       Callback registered successfully, or the callback is already registered.
+ * @retval -EFAULT The provided cb function was \c NULL.
```

Comment:
To align with `pdn_default_ctx_cb_dereg` doxygen.
```suggestion
 * @retval -EFAULT The provided cb parameter was \c NULL.
```